com.unity.render-pipelines.universal/Runtime/Debug/DebugHandler.cs

```diff
@@ -281,7 +279,7 @@ internal void Setup(ScriptableRenderContext context, ref CameraData cameraData)
             }
 
             context.ExecuteCommandBuffer(cmd);
-            CommandBufferPool.Release(cmd);
```

Comment:
similar question as before: are ExecuteCommandBuffer and Clear needed because of cmd buffer syncing?

Reply:
will investigate this